README.md

```diff
@@ -68,7 +68,7 @@ if you're using Lua for your configs. Read the documentation below to learn
 more about specific options (*NOTE:* options in the `setup` function don't
 have the `gitblame_` prefix).
 
-> **NOTE:** you don't have to call `setup` if you don't want to customize the
+> **NOTE:** you must call `setup` if even if you don't want to customize the
```

Comment:
Would calling `setup` on the lib side once be an issue?

Reply:
yes. `enabled` option is creating issue.

Reply:
🤔
What if we disable in setup if `enabled` is false?

Reply:
That disable call is not clearing the virtual text.

Here is what's happening -
Default is `enabled=true` when we run `setup()` in `plugin/gitblame.lua` it enabled the plugin.
When user calls `setup()` with `enabled=false`, `setup()` calls `disable()` which calls `clear_all_extmarks()`.
But `clear_all_extmarks()` is not finding all buffers for some reason and virtual text for buffer is not cleaned.

Reply:
Hmm, this sounds like a bug :)

Reply:
It was problem with `show_blame_info()`. 
1st `setup()` calls `show_blame_info` as callback of vim.schedule(). \
Which is asynchronous call, and takes some time to reach callback. 
So 2nd `setup()` call with `enabled=false` executes `disable()` before `show_blame_info`.
Which is why blame info was showing for one line.

It is fixed with latest commit

Reply:
Now that the issue is resolved, would we be able to disable the plugin in setup if `enabled` is false? Ideally we want to try to maintain backwards compatibility as much as possible, and requiring `setup` might cause friction for users updating the plugin :slightly_smiling_face:

Reply:
Yep. It works!